Docs/Configuration.tex

```diff
@@ -601,16 +601,14 @@ \subsection{Installation and Upgrade}\label{configuration-install}
 \href{https://github.com/corpnewt/ProperTree}{ProperTree} editor can be
 utilised.
 
-It is strongly advised not to use any software that is aware of the internal
-configration structure as it constantly gets out of date and will cause
-incorrect configuration to be generated. If it is a must desprite the
-warning one should make sure to only use stable versions of OpenCore
-with explicit support for the particular version in the app. The choice
-of open-source implementations with transparent binary generation
-is encouraged (e.g. \href{https://github.com/ic005k/QtOpenCoreConfig}{OCAT}),
-since other tools may contain malware. Remember that a configuration
-made for a different hardware setup shall never be used on another hardware
-setup.
+It is strongly advised to avoid software that is aware of the internal configuration
+structure as this may result in invalid configurations given that the structure is
+constantly updated. If such software is to be used despite this warning, ensure that
+only stable versions of OpenCore explicitly supported by such software are used. In
+such cases, open-source implementations with transparent binary generation (such as
+\href{https://github.com/ic005k/QtOpenCoreConfig}{OCAT}) should used given that other
```

Comment:
should _be_ used?

Reply:
Sorry ... pushed a change but then realised you must have been referring to some other part which I can't locate.

Reply:
Found it on the last line above!
Will change